workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/SpatialOSComponent.cs

```diff
@@ -36,5 +36,24 @@ public bool TryGetGameObjectForSpatialOSEntityId(EntityId entityId, out GameObje
             linkedGameObject = entityManager.GetComponentObject<GameObjectReference>(entity).GameObject;
             return true;
         }
+
+        public bool TryGetSpatialOSEntityIdForGameObject(GameObject linkedGameObject, out EntityId entityId)
+        {
+            var component = linkedGameObject.GetComponent<SpatialOSComponent>();
+            if (component == null)
+            {
+                entityId = default(EntityId);
+                return false;
+            }
+
+            if (Worker != component.Worker)
+            {
+                entityId = default(EntityId);
```

Comment:
Small nit - you can call `default(EntityId)` at the start of the function and just return false in these two ifs